NUITizenGallery/Examples/AbsoluteLayoutTest/AbsoluteLayoutTestPage.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="UTF-8" ?>
+<ContentPage x:Class="NUITizenGallery.AbsoluteLayoutTestPage"
+  xmlns="http://tizen.org/Tizen.NUI/2018/XAML"
+  xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+  WidthSpecification="{Static LayoutParamPolicies.MatchParent}"
+  HeightSpecification="{Static LayoutParamPolicies.MatchParent}">
+
+    <ContentPage.AppBar>
+        <AppBar x:Name="appBar" Title="AbsoluteLayoutTestPage"/>
+    </ContentPage.AppBar>
+
+    <ContentPage.Content>
+        <View x:Name="ContentView"
+          WidthSpecification="{Static LayoutParamPolicies.MatchParent}"
+          HeightSpecification="{Static LayoutParamPolicies.MatchParent}">
+            <TextLabel Text="Stylish Header" PointSize="12" Position2D="73,130"/>
+            <View BackgroundColor="#FF0000" Position2D="32,108" Size2D="400,5"/>
+            <View BackgroundColor="#00FF00" Position2D="32,120" Size2D="400,5"/>
+            <View BackgroundColor="#0000FF" Position2D="42,100" Size2D="5,100"/>
+            <View BackgroundColor="#FF00FF" Position2D="54,100" Size2D="5,100"/>
+        </View>
+    </ContentPage.Content>
+
+</ContentPage>
```

Comment:
could you add new line at the end of file from next time? (for this time, I approve~)
reference: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
I'll update :)